driver/web/conversions_application.go

```diff
@@ -80,6 +80,7 @@ func applicationPermissionToDef(item model.Permission) Def.Permission {
 	if assigners == nil {
 		assigners = make([]string, 0)
 	}
+	assigners = append(assigners, model.PermissionGrantAllPermissions)
```

Comment:
I don't think we need to append this here since this can be assumed across the system. Let me know if I am missing the reason this is needed.

Reply:
The main reason I added this is to avoid hardcoding the name of the default assigner permission in the admin app (or other system components), since it tries to determine what a user can assign based on assigners lists.